### PR TITLE
Fixed travis build.

### PR DESCRIPTION
**Pull request by griswaldbrooks**
There were many things, but now it builds.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,10 +8,68 @@ cache:
...
+    # Prepare rosdep to install dependencies.
+    - sudo rosdep init
+    - rosdep update
+    # Install platformio
     - pip install platformio
+    ### Get platformio libs.
+    # Install Timer.h library for easy ISRs.
+    # http://platformio.org/lib/show/75/Timer
+    # Pull in custom IMU libraries that use the i2c_t3.h instead of Time.h
+    - platformio lib -g install 75 https://github.com/victoriarobotics/lsm6-arduino.git https://github.com/victoriarobotics/lis3mdl-arduino.git
```
Comment (markwomack) [inline .travis.yml]: What about lis3mdl-arduino and Timer repositories?  Those are not needed for a green build?

Comment (griswaldbrooks) [reply .travis.yml]: I don't understand. You're commenting on the line where they are installed.
Though I understand the syntax is a bit unfamiliar but in platforimio speak, `75` is the library number, where `75` is `Timer.h`

Comment (markwomack) [reply .travis.yml]: Sorry, I missed the wrap around the first time around. I don't pretend to understand all the Travis stuff, but I see it is including the right libraries here at least.

**Comment by markwomack** [comment]
Approved

**Comment by markwomack** [comment]
There should be a "merge and close" option in github.

**Pull request merged**